app/javascript/packs/application.js

```diff
@@ -1,9 +1,11 @@
 import main from '../mastodon/main';
 
 if (!window.Intl || !Object.assign || !Number.isNaN ||
-    !window.Symbol || !Array.prototype.includes) {
+  !window.Symbol || !Array.prototype.includes) {
   // load polyfills dynamically
-  import('../mastodon/polyfills').then(main);
+  require('intl');
+  require('intl/locale-data/jsonp/en.js');
+  import('../mastodon/polyfills').then(main());
```

Comment:
I'm confused, because without explicitly calling `main()`, main _never gets called _ in Safari 9. `import()` is a cool function, is there something about Safari 9 which does not support it? Mastodon was loading in Safari 9 before this change. I will read through the blog post again, thanks.